### ISSUE TITLE
after signing CLA user reaches a dead end

### ISSUE THREAD

**Issue by bradcupit** [sonatype-nexus-community/salesforce-cla#2]
After going through the CLA license agreement I end up on this page with nowhere else to go.

<img width="1636" alt="screen shot 2018-04-19 at 3 48 08 pm" src="https://user-images.githubusercontent.com/106213/39014875-8e83e366-43e9-11e8-8506-594a7c28be00.png">

Ideally the page would have a link back to the original PR.

**Comment by DarthHater** [comment]
Yeah, totally! @jamesward , I think this is a good idea. Maybe we can figure it out together?

**Comment by jamesward** [comment]
Yup!  Let's do it.  Should be pretty easy.

**Comment by jamesward** [comment]
This has been implemented: https://github.com/salesforce/dr-cla/commit/a5b018f916b958711b2cdd602ad3caaf5fd53e15
